PlanetNode/GraphTypes/ApplicationQuery.cs

```diff
@@ -14,8 +14,9 @@ public ApplicationQuery(BlockChain<PolymorphicAction<PlanetAction>> blockChain)
     {
         Field<StringGraphType>(
             "asset",
+            description: "Queries address's asset(The unit of asset(balance) amount is PNG.)",
             arguments: new QueryArguments(
-                new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "address" }
+                new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "address", Description = "Argument adress is public key. That is like account holder." }
```

Comment:
As the line becomes too long (> 100 characters), we'd better to separate it into multiple lines.  It's natural to give a line to each field assignment.  (Note that the trailing comma of the last field assignment is optional, but it's left for consistency.  It's preferred style in the .NET community.)

Also, it's already a parameter, IMHO the first word “argument” is redundant.

```suggestion
                new QueryArgument<NonNullGraphType<StringGraphType>>
                {
                    Name = "address",
                    Description = "The account holder's 40-hex address",
                }
```